web/src/cypress/support/graphql.ts

```diff
@@ -5,11 +5,8 @@ interface GraphQLResponse {
 }
 
 // runs a graphql query returning the data response (after asserting no errors)
-function graphql(
-  query: string,
-  variables?: any,
-  url = '/v1/graphql',
-): Cypress.Chainable<any> {
+function graphql2(query: string, variables?: any): Cypress.Chainable<any> {
```

Comment:
What do you think of keeping the name as `graphql`? I think in the very near future we will be able to start stripping away code like this and have a single source of truth rather than `graphql` and `graphql2` constantly.

That would cause a bit of bloat in this PR since we would have to update references like `cy.graphql2()` to `cy.graphql()`, but I think it's worth it. I'd be happy to help, lemme know your thoughts